test/unit/cli/opts.test.js

```diff
@@ -199,4 +167,29 @@ test('Options', async t => {
 
 function argv(... args) {
   return ['node', 'ez-build', ... args]
+}
+
+function generateCombinations(flags) {
```

Comment:
Almost, not all permutations are added. Specifically, not all different orders are tested, since the set balloons in size and the test takes too long (still sub-sec, but not by much.)